test/e2e_node/conformance_test.go

```diff
@@ -115,12 +108,23 @@ var _ = Describe("Container Conformance Test", func() {
 				err := terminateCase.Create()
 				Expect(err).NotTo(HaveOccurred())
 
-				// TODO: Check that the container enters running state by sleeping in the container #23309
+				for start := time.Now(); time.Since(start) < retryTimeout; time.Sleep(pollInterval) {
```

Comment:
This seems to just check that the container stops being pending, but that doesn't ensure that container enters running.   It could have failed without entering running.  Is this necessary given the "Eventually" section following this?